srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -329,7 +331,10 @@ private void retrieveAdlNotes(final int dfn, final Patient patient)
                 
                 final ReferenceNotes allNotes = (ReferenceNotes) unmarshaller.unmarshal(input);
```

Comment:
On second look, I find `allNotes` a confusing variable name here. It implies that this is all notes VistA has for the patient.